eng/policheck_exclusions.xml

```diff
@@ -10,6 +10,8 @@
   <!--<Exclusion Type="FileName">ABC.TXT|XYZ.CS</Exclusion>-->
 
   <Exclusion Type="FolderPathFull">.DOTNET</Exclusion>
+  <!-- This file is intentionally used in the unit tests. -->
+  <Exclusion Type="FileName">MYCERT.PFX</Exclusion>
```

Comment:
May I clarify if the policheck_exclusions file changes will affect the credscan as well?

Reply:
No, it  won't. Actually I misunderstood it. Thanks for pointing it out!